client.go

```diff
@@ -1047,6 +1047,22 @@ func (c *Client[TTx]) jobCancel(ctx context.Context, exec riverdriver.Executor,
 	})
 }
 
+// JobDelete deletes the job with the given ID from the database, returning
+// whether or not the job was deleted along with a possible error. Jobs in the
+// running state are not deleted.
+func (c *Client[TTx]) JobDelete(ctx context.Context, id int64) (bool, error) {
```

Comment:
Thoughts on returning the deleted job row instead of a bool? Feels like it gives you gives you more information and not a lot of downside.

Reply:
I think that will work if we also make the change to return an error when the job is running, I like that better.